en/08_Changelogs/6.0.0.md

```diff
@@ -1139,6 +1152,50 @@ SilverStripe\ORM\Connect\MySQLDatabase:
   collation: utf8mb4_unicode_ci
 ```
 
+### TinyMCE is in a separate module now {#tinymce}
+
+> [!WARNING]
+> To continue using TinyMCE as an HTML editor in your existing project, you must add [`silverstripe/htmleditor-tinymce`](https://packagist.org/packages/silverstripe/htmleditor-tinymce) as an explicit dependency.
+
+Silverstripe CMS 6 continues to use TinyMCE 6, which was the same version used for CMS 5. This is because TinyMCE changed its license to `GPLv2+` in TinyMCE 7. That license isn't compatible with our `BSD-3` licenses, so we made the decision to not upgrade our TinyMCE dependency.
+
+Instead, we have moved all TinyMCE functionality used by Silverstripe CMS into a new module - `silverstripe/htmleditor-tinymce`. To continue using TinyMCE in your project, you must add that dependency in your `composer.json` file.
+
+Part of this change includes further abstracting the [`HTMLEditorConfig](api:SilverStripe\Forms\HTMLEditor\HTMLEditorConfig) class and related functionality, which makes it easier to replace TinyMCE with an alternative HTML editor implementation. We intend to replace TinyMCE as the default HTML editor for new projects with an alternative editor in a future minor release of Silverstripe CMS. Existing projects will be able to opt-in to the new editor when it is available, and all projects including new projects will be able to continue using TinyMCE for the lifetime of CMS 6.
+
+Part of this change includes a new way to define rules such as which elements and attributes are allowed in HTML content. You can continue to set the `valid_elements` and `extended_valid_elements` configuration if you want to, but the new API is generic and will work for any `HTMLEditorConfig` implementation and can be defined in YAML configuration.
+
+See [defining HTML editor configurations](/developer_guides/forms/field_types/htmleditorfield/#defining-html-editor-configurations) for details about the new generalised API, and [optional features > TinyMCE HTML editor](/optional_features/htmleditor-tinymce/) for any TinyMCE-specific documentation.
+
+#### If you want TinyMCE 7 anyway
+
+If you want to use TinyMCE 7 in your own projects, and have done your due diligence to validate that the new license won't cause you any legal problems, you can theoretically do that.
+
+> [!WARNING]
+> Using TinyMCE 7 with Silverstripe CMS isn't supported and has not been tested.
+>
+> The plugins, skin, and configuration that come with `silverstripe/htmleditor-tinymce` are not guaranteed to work with TinyMCE 7.
```

Comment:
This is a question that people are likely to ask - I think it's best to just answer it outright and say it's theoretically doable, but not supported.